Formula/yaz.rb

```diff
@@ -34,17 +34,20 @@ class Yaz < Formula
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
+    depends_on "tclsh" => :build
```

Comment:
`tclsh` is not a formula, so this is not a valid dependency. Do you mean `tcl-tk`?

Reply:
Just needs tclsh. If tcl-tk has that, fine.